backend/commons/src/main/java/org/sonarsource/sonarlint/core/commons/util/git/BlameParser.java

```diff
@@ -73,8 +75,19 @@ public static SonarLintBlameResult parseBlameOutput(String blameOutput, String c
     return new SonarLintBlameResult(blameResult, projectBaseDir);
   }
 
-  public static int countCommitterTimeOccurrences(String blameOutput) {
-    var pattern = Pattern.compile("^" + COMMITTER_TIME, Pattern.MULTILINE);
+  private static boolean shouldSkipSection(String fileSection, boolean lineNumberIsOff) {
+    if (fileSection.isBlank()) {
+      return true;
+    }
+    if (lineNumberIsOff) {
+      LOG.warn("Number of blame output sections is higher than expected number of lines. Section content: {}", fileSection);
```

Comment:
We could manually create an exception here to log it, so we had a visibility of such events on Sentry. It could help to understand better what's going on.